contracts/access/AccessControl.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-// OpenZeppelin Contracts (last updated v4.9.0-rc.0) (access/AccessControl.sol)
+// OpenZeppelin Contracts (last updated v4.9.0-rc.1) (access/AccessControl.sol)
```

Comment:
Even though there were no changes in this file, this is updated as a consequence of this line:

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/3ec4307c8a79ddfee9bbaf1f26435c908fae0cae/scripts/release/update-comment.js#L19

I think we put that line in so that the comment gets updated from `v4.9.0-rc.0` to `v4.9.0` in the final release, but for intermediate prereleases I feel we should only update what has changed.

@ernestognw @Amxx Any thoughts?

Reply:
We can apply this only if it's in pre-release mode. We need to look for `pre.json` with mode `pre`, as seen [here](https://github.com/changesets/changesets/blob/8ede784e5fc3629858a57643fdaab76ffb631520/packages/types/src/index.ts#LL167C8-L167C8).

Reply:
I think the logic can be implemented without looking at the pre.json, which might be simpler.

Reply:
I don't clearly see how, can you elaborate?

Reply:
I believe it should be as simple as sorting like `v4.8.2 < v4.8.3 < v4.9.0 < v4.9.0-rc.0 < v4.9.0-rc.1`.

Reply:
See https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4244.